### ISSUE TITLE
How to get or generate the mean_image?

### ISSUE THREAD

**Issue by catmonkeylee** [dmlc/mxnet#604]
When I read the notebook 'Use Pretrained Inception-BatchNorm Network',  I don't know how to get the mean_image :
# load mean image
mean_img = mx.nd.load("Inception/mean_224.nd")["mean_img"]

It was generated when train the imagenet? When I read the alexnet, I can not find the code that generated the  mean_img.   If I used my images to train the model, How to generate the mean image.

**Comment by austingg** [comment]
The image iterator will automatically calculate/load mean file if you set mean_file="xxxxx".

**Comment by catmonkeylee** [comment]
Thanks, I read the source code, in train process the mean_file path is not exists, so will automatically caculate the file and save the mean.  in test process the mean_file path is exists, so will load and use the mean.

**Comment by ysh329** [comment]
Thanks a lot!

**Comment by mcxkarakoc** [comment]
subscribe
